### PR TITLE
Update examples for events using interfaces

### PR DESCRIPTION
**Pull request by sizeg**
Where was a mistake in example for events using interfaces. As events has no method [[className()]] and when called [[Event::trigger()]] we should pass first argument class name implemented this interface, not the interface itself.

### CODE REVIEW

File: docs/guide/concept-events.md [hunk 1]
```diff
@@ -295,13 +295,22 @@ pass the interface name as the first argument:
 ```php
 Event::on('DanceEventInterface', DanceEventInterface::EVENT_DANCE, function ($event) {
     Yii::trace($event->sender->className . ' just danced'); // Will log that Dog or Developer danced
-})
+});
+
+// or with fully qualified interface name
+Event::on(DanceEventInterface::class, DanceEventInterface::EVENT_DANCE, function ($event) {
```
Comment (cebe) [inline docs/guide/concept-events.md]: `::class` does not work on php 5.4 so it should not be used in 2.0.x docs.

**Review by cebe — CHANGES_REQUESTED** [review]

**Comment by samdark** [comment]
It should pass interface name, I think. W/o using `::className()`, of course.

**Comment by sizeg** [comment]
> It should pass interface name

@samdark then it will conflict with the next paragrath
```php
// DOES NOT WORK
Event::trigger('DanceEventInterface', DanceEventInterface::EVENT_DANCE); // error
```

**Comment by sizeg** [comment]
We should pass to [[Event::trigger()]] a fully qualified class name implemented our event interface. By the way I'm using it this way.

According to the previouse examples from doc, were two new objects created from Component, we can use [[className()]] I think.

**Comment by samdark** [comment]
@cebe need your assistance in this one...

**Review by dynasource — APPROVED** [review]
these updates are correct

**Comment by SilverFire** [comment]
Resolved by f1f357b

**Comment by SilverFire** [comment]
@sizeg thank you for noticing

**Pull request closed without merging**